.changeset/tender-ghosts-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+"@replayio/sourcemap-upload": patch
+---
+
+Update `sourcemap-upload `package to support deletion and correlating sourcemaps by filename
```

Comment:
This is going to be part of the CHANGELOG of this `@replayio/sourcemap-upload` package. So it should be written from its PoV. Mentioning "sourcema-upload" here is redundant. 

You could also mention what kind of options where added. It's helpful to know what's going to be possible now but some extra references to *how* would be useful too